scripts/01_checkpoints_to_stats/compute_stats.py

```diff
@@ -0,0 +1,49 @@
+from transformers import (
+    AutoModelForMaskedLM,
+    # AutoModelForSequenceClassification,
+    BertForSequenceClassification,
+    AutoTokenizer,
+    DataCollatorForLanguageModeling,
+)
+from datasets import load_dataset, load_from_disk, Dataset
+
+from torch.utils.data import DataLoader
+import os
+from src.metrics import get_metrics_hf_transformer, get_lm_loss_hf_transformer
+from tqdm import tqdm
+import json
+import argparse
+import glob
+
+
+def write_stats_to_file(model, out_dir, epoch, train_dataloader=None, device="cuda"):
+
+    data = get_metrics_hf_transformer(model)
+
+    # make directory if it does not exist
+    if not os.path.exists(out_dir):
+        os.makedirs(out_dir)
+
+    with open(os.path.join(out_dir, f"stats_{epoch}_losses"), "w") as f:
+        print(f)
+        json.dump(data, f)
+
+
+def main(out_dir, model_dir=None):
+    checkpoints = glob.glob(model_dir + "*/")
+
+    for d in tqdm(checkpoints):
+        model = BertForSequenceClassification.from_pretrained(d)
+
+        epoch = int(d.split("-")[-1][:-1])
+
+        print(epoch)
+        write_stats_to_file(model, out_dir, epoch)  
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--model_dir", type=str)
```

Comment:
Should add descriptions (using the `help` parameter) for these arguments - especially since it seems like this script expects that this directory contains only checkpoint sub-directories, and they have to be formatted a very specific way.